### ISSUE TITLE
Embed Instructions and Generated Markup

### ISSUE THREAD

**Issue by netpoetica** [stolksdorf/homebrewery#661]
## Description

@stolksdorf Firstly, just FYI, this tool is amazeballs - great work.

In addition to [this single CSS file](https://github.com/stolksdorf/homebrewery/issues/10), it would be great if there was an "Embed" button on Homebrewery that gave a popup including:

- instructions to download that CSS file
- a style tag w/ that file and instructions to put it in your pages head
- an HTML version of what was created that can easily be copied to clipboard

What do you think?

The use case is, an author wants to host the content on their own site but doesn't want it to be a PDF. Very likely, in my case, I would still generate a PDF w/ a download like (and usually a link to Homebrewery next to it, in case people want the errata), but it would be great to show something inline on my own page.

I understand that having the CSS file is enough, but it would be useful to add a quick "EMBED </>" link to the Homebrewery content so the user could host it themselves.

**Comment by ericscheid** [comment]
See also #956

**Comment by 5e-Cleric** [comment]
Would it be agreeable to give an option to download as a zip file containing the HTML and CSS files? or embedding the styles into the HTML then export as that? I can work on this feature but we should add this as a toggle in accountpage, or a pick a brew to download like this, as i don't think this is going to be a popular feature.

**Comment by dbolack-ab** [comment]
This and the related issues should be closed and reopened with a cosolidated feature plan.

**Comment by 5e-Cleric** [comment]
Agreed, we should close this and put an issue with all the desired output formats or really consider if this is something we want, which i think not, honestly
